modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java

```diff
@@ -412,6 +412,18 @@ public void testValidateSingleApproverScriptedAssignmentDefinition()
 		_assertValid(inputStream);
 	}
 
+	@Test
+	public void testValidateTaskWithMoreThanOneTransitions() throws Exception {
+		InputStream inputStream = _getResourceInputStream(
```

Comment:
Hey @seltong,

rename to `testValidateTaskWithMoreThanOneDefaultTransition`. please